.github/workflows/ci.yml

```
@@ -1,75 +1,95 @@
-name: Full Service CI
+name: ci
 
 env:
-  DOCKERHUB_REPO: mobilecoin/full-service
+  SGX_MODE: HW
+  IAS_MODE: PROD
+  RUST_BACKTRACE: full
+  CONSENSUS_ENCLAVE_CSS: /var/tmp/consensus-enclave.css
+  INGEST_ENCLAVE_CSS: /var/tmp/ingest-enclave.css
 
 on:
-  push:
-    tags:
-      - 'v*'
+  pull_request:
     branches:
-      - 'develop'
-      - 'feature/**'
+      - develop
+      - main
```

I imagine we want to enable this events other than PRs, since the steps check for `startsWith(refs/tags/v)`.
```suggestion
  pull_request:
  push:
    tags: v*
    branches:
      - master
      - main
      - develop
  workflow_dispatch:
```

---

what other situation would you see this being useful besides PRs?